packages/cli/test/functional/testUtil/compare.js

```diff
@@ -17,7 +17,7 @@ const TEST_BLACKLIST = ignore().add([
 ]);
 
 // Files that possibly have null characters but are not binary files
-const NULL_WHITELIST = ignore().add(['components.min.js']);
+const NULL_WHITELIST = ignore();
```

Comment:
Could the `NULL_WHITELIST` be removed entirely?

Reply:
Yep it can be! I tried removing it and making the related changes in the rest of the file and it passes all test cases. I left it as is for the ease of addition of future files having null characters. How do you feel about this?

Reply:
> Yep it can be! I tried removing it and making the related changes in the rest of the file and it passes all test cases. I left it as is for the ease of addition of future files having null characters. How do you feel about this?

Let's remove it then, future updates can reference the history to find out how to do so.

Reply:
Alright! I have removed it in the latest commit.